src/CodeGen_LLVM.cpp

```diff
@@ -436,9 +432,7 @@ llvm::Module *CodeGen_LLVM::compile(const Module &input) {
     CodeGen_LLVM::optimize_module();
 
     // Disown the module and return it.
-    llvm::Module *m = module;
-    module = NULL;
-    return m;
+    return std::move(module);
```

Comment:
Pretty sure that it's always redundant to use std::move() for a returned value (and in fact, I think Google's build system will warn if you do)